README.md

```diff
@@ -1,273 +1,247 @@
-| [weaver registry update-markdown](docs/usage.md#registry-update-markdown) | Update semconv snippet-based markdown files |
-| [weaver registry stats](docs/usage.md#registry-stats)                     | Generate statistics on a semconv registry   |
+![Observability by Design Workflow - Validate](docs/images/workflow_validate.svg)
 
-Phase 2 will introduce commands related to the management of Application
-Telemetry Schemas.
+### Policy-based Static Validation Check
 
-## Documentation
+Prevent breaking changes and enforce standards:
 
-- [Weaver Architecture](docs/architecture.md): A document detailing the architecture of the project.
-- [Weaver Configuration](docs/weaver-config.md): A document detailing the configuration options available.
-- [Weaver Forge](crates/weaver_forge/README.md): An integrated template engine designed to generate
-  documentation and code based on semantic conventions.
-- [Weaver Checker](crates/weaver_checker/README.md): An integrated policy
-  engine for enforcing policies on semantic conventions.
-- [Schema Changes](docs/schema-changes.md): A document describing the data model
-  used to represent the differences between two versions of a semantic convention registry.
-- [Application Telemetry Schema OTEP](https://github.com/open-telemetry/oteps/blob/main/text/0243-app-telemetry-schema-vision-roadmap.md):
-  A vision and roadmap for the concept of Application Telemetry Schema.
-- Presentation slides from the Semantic Convention SIG meeting on October 23,
-  2023 [here](https://docs.google.com/presentation/d/1nxt5VFlC1mUjZ8eecUYK4e4SxThpIVj1IRnIcodMsNI/edit?usp=sharing).
-
-## Examples of Code and Documentation Generation
-
-The following examples show how to generate code and documentation for various languages using Weaver.
-
-> Note: A more out of the box experience will be available in the future.
-
-**Java**
-From GitHub repo main branch.
+- **Verify best practices compliance**
+- **Validate schema changes (avoid breaking alerts, dashboards, ...)**
 
 ```bash
-weaver registry generate \
---templates https://github.com/open-telemetry/semantic-conventions-java.git[buildscripts/templates] \
-java
+weaver registry check --policy policies/
 ```
 
-**C++**
-From a specific release using the corresponding archive.
-
-```bash
-weaver registry generate \
---templates https://github.com/open-telemetry/opentelemetry-cpp/archive/refs/tags/v1.20.0.zip[buildscripts/semantic-convention/templates] \
-./ \
---param filter=any \
---param output=output/ \
---param schema_url=https://opentelemetry.io/schemas/v1.32.0
+```
+Violation: Metric 'auction.bid.count' cannot change
+required/recommended attributes (missing '{"auction.
+id"}')
+- Category        : backward_compatibility
+- Type           : semconv_attribute
+- SemConv group  : metric.auction.bid.count
+- SemConv attribute: auction.id
+- Provenance: /home/weaver/source
+
+Total execution time: 5.531342577s
+make: *** [Makefile:89: check] Error 1
 ```
 
```

Comment:
A slide that I had in the SRECON presentation but didn’t include in the Kubecon presentation for timing reasons could be interesting to share here.

It's a summary of the policies we already have in place. We could organize the following list as a 2 columns table.

- No attributes outside registry.
- Definitions require stability.
- No requirement levels on attributes
- Names must follow format rules
- IDs must match naming patterns
- Attributes fully qualified
- No constant name collisions
- No namespace collisions
- No duplicate attributes in group
- Experimental attributes in stable groups must be opt-in
- No removal of elements
- No stability downgrades
- No type or unit changes
- Attribute sets immutable
- Enum values immutable